Project.toml

```diff
@@ -70,6 +71,7 @@ Tables = "1"
 TruncatedStacktraces = "1"
 ZygoteRules = "0.2"
 julia = "1.6"
+DiffEqBase = "6.132.0"
```

Comment:
why is this required?

Reply:
Eh I thought you asked for this so that the `progress_id` is guaranteed to work.

https://github.com/SciML/SciMLBase.jl/pull/514#discussion_r1341953237

Reply:
Is that not what you wanted?